app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -41,7 +46,12 @@ def google_billing
     begin
       provider = request.env["omniauth.auth"].dig('provider')
       self.class.validate_scopes_from_params(params, provider)
+      self.class.validate_host_header_on_callback(request.headers)
     rescue SecurityError => e
+      logger.error e.message
+      context = ErrorTracker.format_extra_context(params)
+      ErrorTracker.report_exception(e, @user, context)
```

Comment:
I think making a new `ErrorTracker.report_exception_formatted` that combines these two steps would simplify this code, and make the interface clearer

Reply:
I think that's a nice enhancement, but that also touches 50+ places, so I'd rather ticket that work and do it separately from this so that we don't hold up this merge.

Reply:
Tracked here: https://broadworkbench.atlassian.net/browse/SCP-3378